pkg/controller/launch/pool_manager.go

```diff
@@ -74,12 +81,29 @@ func (cpm *ConcretePoolManager) GetStatus() (status *PoolStatus, err error) {
 }
 
 func (cpm *ConcretePoolManager) SetStatus(status *PoolStatus) error {
+	var healthy, schedulable int64
```

Comment:
It would be cool to break out the `healthy` and `schedulable` fields into the `PoolStatus`. That would allow us to do things like self-healing and make the health status known to the controller too.